.github/workflows/bot.yml

```diff
@@ -705,17 +705,23 @@ jobs:
       matrix:
         include:
           - flinkProfile: "flink1.15"
-            flinkAvroVersion: "1.10.0"
```

Comment:
For example Flink avro Version already update to 1.11.4 based on pom. But in bot.xml it's still 1.10.0